File: Sources/gestion-culturelle-back/src/main/java/com/afpa/cda/dto/SalleDto.java
package com.afpa.cda.dto;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class SalleDto {

	private int id;
	private String label;
	private int capacite;
	private int placesVip;
	private int fraisJournalier;
	private TypeSalleDto typeSalle;
	//private List<ManifestationDto> manifestations;

}

File: Sources/gestion-culturelle-front/project/src/app/model/salleDto.ts
import { TypeSalleDto } from '../model/typeSalleDto';


export class SalleDto {
    id: number;
    label: string;
    capacite: number;
    placesVip: number;
    fraisJournalier: number;
    typeSalle:TypeSalleDto;

    constructor(id?: number, 
        param1?: string, 
        param2?: number,
        param3? : number, 
        param4? : number,
        typeSalle? : TypeSalleDto ){
        this.id = id;
        this.label=param1;
        this.capacite = param2;
        this.fraisJournalier = param3;
        this.fraisJournalier = param4;
        this.typeSalle = typeSalle;
    }
}
